### ISSUE TITLE
ValueError when using rclone.hash()

### ISSUE THREAD

**Issue by alicethefemme** [Johannes11833/rclone_python#42]
Hello,
I am currently facing an issue where I am computing the hashes for every file in a bucket on my Wasabi remote directory. The hash command was working properly without errors on my local machine for testing, with the same remote linked up.
However, when it came to the production machine running my code, the error is there.

Error trace:
```
Traceback (most recent call last):
  File "/root/rclone-backup-filechecker-python/main.py", line 145, in <module>
    main()
  File "/root/rclone-backup-filechecker-python/main.py", line 43, in main
    dict_of_remote = get_hashes_from_files_remote(remote_path)
  File "/root/rclone-backup-filechecker-python/main.py", line 53, in get_hashes_from_files_remote
    result = rclone.hash(HashTypes.md5, remote_path)
  File "/usr/local/lib/python3.10/dist-packages/rclone_python/rclone.py", line 24, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/rclone_python/rclone.py", line 507, in hash
    value, key = l.split()
ValueError: too many values to unpack (expected 2)
```

My code: https://gitlab.atfdev.co.uk/alicethefemme/rclone-backup-filechecker-python

Does anyone know why this might be happening? Thanks!

**Comment by alicethefemme** [comment]
Upon further inspection of the error, the line 522 in rclone.py is the issue. Not sure what is causing this error, but will have to check.

**Comment by alicethefemme** [comment]
I managed to fix this using #43 to fix the error. The pull request passed without any other errors in CI/CD
